qiita_db/support_files/patches/33.sql

```diff
@@ -0,0 +1,435 @@
+
+-- Artifact filepath table - relates an artifact with its files
+CREATE TABLE qiita.artifact_filepath (
+    artifact_id          bigint  NOT NULL,
+    filepath_id          bigint  NOT NULL,
+    CONSTRAINT idx_artifact_filepath PRIMARY KEY ( artifact_id, filepath_id )
+ ) ;
+CREATE INDEX idx_artifact_filepath_artifact ON qiita.artifact_filepath ( artifact_id ) ;
+CREATE INDEX idx_artifact_filepath_filepath ON qiita.artifact_filepath ( filepath_id ) ;
+ALTER TABLE qiita.artifact_filepath ADD CONSTRAINT fk_artifact_filepath_artifact FOREIGN KEY ( artifact_id ) REFERENCES qiita.artifact( artifact_id )    ;
+ALTER TABLE qiita.artifact_filepath ADD CONSTRAINT fk_artifact_filepath_filepath FOREIGN KEY ( filepath_id ) REFERENCES qiita.filepath( filepath_id )    ;
+
+-- Parent artifact table - keeps track of the procenance of a given artifact.
+-- If an artifact doesn't have a parent it means that it was uploaded by the user.
+CREATE TABLE qiita.parent_artifact (
+    artifact_id          bigint  NOT NULL,
+    parent_id            bigint  NOT NULL,
+    CONSTRAINT idx_parent_artifact PRIMARY KEY ( artifact_id, parent_id )
+ ) ;
+CREATE INDEX idx_parent_artifact_artifact ON qiita.parent_artifact ( artifact_id ) ;
+CREATE INDEX idx_parent_artifact_parent ON qiita.parent_artifact ( parent_id ) ;
+ALTER TABLE qiita.parent_artifact ADD CONSTRAINT fk_parent_artifact_artifact FOREIGN KEY ( artifact_id ) REFERENCES qiita.artifact( artifact_id )    ;
+ALTER TABLE qiita.parent_artifact ADD CONSTRAINT fk_parent_artifact_parent FOREIGN KEY ( parent_id ) REFERENCES qiita.artifact( artifact_id )    ;
+
+-- Study artifact table - relates each artifact with its study
+CREATE TABLE qiita.study_artifact (
+    study_id             bigint  NOT NULL,
+    artifact_id          bigint  NOT NULL,
+    CONSTRAINT idx_study_artifact PRIMARY KEY ( study_id, artifact_id )
+ ) ;
+CREATE INDEX idx_study_artifact_study ON qiita.study_artifact ( study_id ) ;
+CREATE INDEX idx_study_artifact_artifact ON qiita.study_artifact ( artifact_id ) ;
+ALTER TABLE qiita.study_artifact ADD CONSTRAINT fk_study_artifact_study FOREIGN KEY ( study_id ) REFERENCES qiita.study( study_id )    ;
+ALTER TABLE qiita.study_artifact ADD CONSTRAINT fk_study_artifact_artifact FOREIGN KEY ( artifact_id ) REFERENCES qiita.artifact( artifact_id )    ;
+
+-- Software table - holds the information of a given software package present
+-- in the system and can be used to process an artifact
+CREATE TABLE qiita.software (
+    software_id          bigserial  NOT NULL,
+    name                 varchar  NOT NULL,
+    version              varchar  NOT NULL,
+    description          varchar  NOT NULL,
+    CONSTRAINT pk_software PRIMARY KEY ( software_id )
+ ) ;
+
+-- soft_command table - holds the information of a command in a given software
+-- this table should be renamed to command once the command table in the
+-- analysis table is merged with this one
+CREATE TABLE qiita.soft_command (
```

Comment:
Changed done. Note that I would prefer to have only command, as the comment reflects, but that needs changes on the analysis pipeline. I think we can rename once the refactor on the analysis pipeline is done.